pkg/agent/api/delegatedidentity/v1/service.go

```diff
@@ -82,24 +82,24 @@ func (s *Service) isCallerAuthorized(ctx context.Context, log logrus.FieldLogger
 		}
 	}
 
-	identities := s.manager.MatchingIdentities(callerSelectors)
-	numRegisteredIDs := len(identities)
+	entries := s.manager.MatchingRegistrationEntries(callerSelectors)
+	numRegisteredIDs := len(entries)
```

Comment:
```suggestion
	numRegisteredEntries := len(entries)
```
the thing being measured has slightly changed